stable/elasticsearch/Chart.yaml

```diff
@@ -1,8 +1,8 @@
 apiVersion: v1
 name: elasticsearch
 home: https://www.elastic.co/products/elasticsearch
-version: 1.28.0
-appVersion: 6.7.0
+version: 1.29.0
```

Comment:
Note that you can't do a rolling update from 6.7 to 7.* .  So should this be a major version bump?  I'm not sure.  But at least there should be some documentation in the readme.

Reply:
@t-d-d thank you for pointing it out! I've added readme and bumped version to `6.8` as I mentioned it in readme as a version supported to perform update 6.8 to 7.1 https://github.com/helm/charts/pull/14535/commits/713768d2d470e89c492e75c2dc7dd6b7ee048c70